### PR TITLE
point to buildpack readme

### PR DESCRIPTION
**Pull request by perrygold**
1.added new section "Use Custom Buildpack"
2.Step 4 :  added sentence to catch users before they issue the cf push command; changed presentation of cf push command to make it stand out better if users are scanning steps.

### CODE REVIEW

File: docs-content/using.html.md.erb [hunk 1]
```diff
@@ -145,7 +156,11 @@ $ cf restage my-app
 On application startup, the service loads the secrets that are defined in the application's `secrets.yml` into the PCF environment. 
 
 
+###<a id='custom-bp'></a>Use a Custom Buildpack
+
+The `cf push` command in the previous procedure defaults to using the buildpacks included in CyberArk Conjur Service Broker for PCF. If your application uses a custom buildpack,  such as a node.js buildpack, see the following file for instructions.
```
Comment (izgeri) [inline docs-content/using.html.md.erb]: - "included in CyberArk ... tile" => "include in the CyberArk ... tile"

- I would remove the "such as a node.js buildpack" comment here - it's more nuanced and complex than that, and probably not worth going into in much detail. It suffices to say that if you are using a custom buildpack or if you need to specify a buildpack in your manifest or at `cf push`, you will need to use this alternate method.

- Instead of linking to the file below, you can change "see the following file for instructions" => "read our [Buildpack documentation](https://github.com/cyberark/cloudfoundry-conjur-buildpack/blob/master/README.md) for instructions."

Comment (perrygold) [reply docs-content/using.html.md.erb]: first  and second done;  on third, we cannot use the word "our" or "we" in the PCF docs (per their style guide,  which makes sense because who is it referring to, PCF or Cyberark?) . Anyway, I said Cyberark Conjur buildpack documentation

File: docs-content/using.html.md.erb [hunk 2]
```diff
@@ -145,7 +156,9 @@ $ cf restage my-app
 On application startup, the service loads the secrets that are defined in the application's `secrets.yml` into the PCF environment. 
 
 
+###<a id='custom-bp'></a>Use a Custom Buildpack
 
+The `cf push` command in the previous procedure defaults to using the buildpacks included in the CyberArk Conjur Service Broker for PCF. If you need to specify any custom buildpacks in the application manifest or in the `cf push` command, see  the [CyberArk Conjur buildpack documentation](https://github.com/cyberark/cloudfoundry-conjur-buildpack/blob/master/README.md) for instructions.
```
Comment (izgeri) [inline docs-content/using.html.md.erb]: This might be clearer if it read something like:

> The CyberArk Conjur Service Broker for PCF uses the Conjur Buildpack to inject secrets into your session environment variables when your application starts. The Conjur Buildpack is a decorator buildpack and will not be properly invoked if your application requires you to specify a buildpack. For applications that must specify a buildpack on `cf push`, please reference the [Conjur Buildpack documentation] for instructions on configuring your application to successfully retrieve secrets from Conjur.

Comment (perrygold) [reply docs-content/using.html.md.erb]: used first two sentences. Good to include info about decorator buildpack.
Question 1: the word "session" is new , potentially confusing.  Can we use "space" there instead? 
re third sentence, I kept mention of the manifest in there; it's not just on cf push, but cf push or in the manifest (as I understood it from Kumbi) . 
Question 2:  In the Install chapter, in the BuildPack tab, we say in absolute terms, don't change the order. Is there any disclaimer needed there about "unless you are using custom buildpack" ?  If so, let's just point to this section where we are discussing the buildpack issue, and state here what might be a valid reason for changing that install ordering. -- can you give me words for that?

Comment (izgeri) [reply docs-content/using.html.md.erb]: 1. It is really injecting the secrets as environment variables into the application process, so that if the process dies the environment variables are removed as well. We're using session as a shorthand here to indicate that the env vars are local to the app process. We could maybe reword as "to inject secrets into your running application process as environment variables when your application starts"
2. I would say "unless you plan to manually deploy the components of the buildpack" - does that answer your question?

Comment (perrygold) [reply docs-content/using.html.md.erb]: re 2... partly... what's the correct settings on that tab if they are doing a manual deploy? do they use 0 on one or both buildpacks?

Comment (perrygold) [reply docs-content/using.html.md.erb]: wait , actually, this is making sense.  They are configuring a tile that should be able to apply to lots of applications.  Most apps use the default BP, but a few use custom BPs.  They can't use the same tile for all?

Comment (izgeri) [reply docs-content/using.html.md.erb]: They can use the same tile for all. They should configure the tile as recommended with meta-buildpack 1st and Conjur buildpack 2nd. Then apps that have to specify a buildpack will do so on `cf push` or in their manifest.

**Comment by izgeri** [comment]
LGTM
